src/app/core/identity-provider/identity-provider.factory.ts

```diff
@@ -28,33 +30,48 @@ export class IdentityProviderFactory {
     type?: string;
   };
 
+  initialized$ = new BehaviorSubject<boolean>(false);
```

Comment:
pls, make this private and write+document a method isInitialized and return this observable